### PR TITLE
Feat: Added .size-limit.js to check all SDKs bundle sizes

### PR DESCRIPTION
**Pull request by saikumarrs**
## Description of the change

Updated size-limit configuration to measure and check all SDKs bundle size.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

N/A

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/533)

### CODE REVIEW

File: .size-limit.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * DO NOT EDIT the size limits configured in this file.
+ * Should you update them, please contact CODEOWNERS.
+ */
+module.exports = [
+  {
+    name: "Core - CDN",
+    path: "dist/rudder-analytics.min.js",
+    gzip: true,
+    limit: "38 kB",
```
Comment (MoumitaM) [inline .size-limit.js:10]: Need to update it to the current value i.e. 38.5 kb

File: .size-limit.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    name: "All Integrations - CDN",
+    path: "dist/integrations/*.min.js",
+    gzip: true,
+    limit: "610 kB",
+  },
+  {
+    name: "Core - NPM",
+    path: "dist/rudder-sdk-js/index.js",
+    gzip: true,
+    limit: "38 kB",
```
Comment (MoumitaM) [inline .size-limit.js:22]: Same here.

File: package.json [hunk 3]
```diff
@@ -22,13 +14,13 @@
...
     "bundle-size-visual": "NODE_ENV=true ENV=prod PROD_DEBUG=true uglify=true visualizer=true rollup -c",
-    "size-build": "npm run buildProdBrowser",
-    "size": "npm run size-build && size-limit",
+    "size": "npm run buildProdBrowser && npm run buildAllIntegrations && npm run buildForNpm && size-limit",
     "changelog": "auto-changelog -p -t keepachangelog -u true -l false --sort-commits date-desc ",
     "buildDevIntegration": "NODE_ENV=false uglify=false INTG_NAME=AdobeAnalytics rollup --config rollup.intgConfig.js",
     "buildDevIntegrationCLI": "NODE_ENV=false uglify=false INTG_NAME=$npm_config_intg rollup --config rollup.intgConfig.js",
     "buildProdIntegration": "NODE_ENV=false ENV=prod uglify=true INTG_NAME=AdobeAnalytics rollup --config rollup.intgConfig.js",
-    "buildProdIntegrationCLI": "NODE_ENV=false ENV=prod uglify=true INTG_NAME=$npm_config_intg rollup --config rollup.intgConfig.js"
+    "buildProdIntegrationCLI": "NODE_ENV=false ENV=prod uglify=true INTG_NAME=$npm_config_intg rollup --config rollup.intgConfig.js",
```
Comment (MoumitaM) [inline package.json:22]: Add `PROD_DEBUG=true` for these commands.

Comment (saikumarrs) [reply package.json:22]: This is already taken care in this PR,
https://github.com/rudderlabs/rudder-sdk-js/pull/537/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

File: .size-limit.js (hunk 2 above)
Comment (saikumarrs) [reply .size-limit.js:22]: We'll update the size limit later during integration.

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**